File: client/infernoapp/src/store.js
///////////////////////////////////////////////////////
//                     xibbit 1.50                   //
//    This source code is a trade secret owned by    //
// Daniel W. Howard and Sanjana A. Joshi Partnership //
//              Do not remove this notice            //
///////////////////////////////////////////////////////
import { applyMiddleware, compose, createStore } from 'redux'
import thunk from 'redux-thunk';
import createRootReducer from './reducers'

export default function configureStore(preloadedState) {
  const store = createStore(
    createRootReducer(),
    preloadedState,
    compose(
      applyMiddleware(thunk)
    )
  )

  return store
}

File: client/flutter_app/lib/modules/store.dart
///////////////////////////////////////////////////////
//                    xibbit 1.50                    //
//    This source code is a trade secret owned by    //
// Daniel W. Howard and Sanjana A. Joshi Partnership //
//             Do not remove this notice             //
///////////////////////////////////////////////////////
import 'package:redux/redux.dart';
import '../reducers/index.dart';

// a single global store
final store = Store<Map>(rootReducer, initialState: initialState);
